recipe/meta.yaml

```diff
@@ -21,22 +21,21 @@ requirements:
     - python
     - setuptools
   run:
-    - numpy >=1.10
-    - pyvisa >=1.8
-    - ipython >=4.1.0
-    - jupyter >=1.0.0
-    - ipywidgets >=4.1
-    - h5py >=2.6
-    - pathlib
-    - matplotlib
-    - coverage
-    - pyqtgraph
+  - jupyter ==1.0.0
+  - numpy ==1.11.3
+  - matplotlib ==2.0.0
+  - pyqtgraph ==0.10.0
+  - PyVISA ==1.8
+  - PyQt5 ==5.7.1
+  - sip ==4.19
+  - QtPy ==1.2.0
+  - h5py ==2.6.0
```

Comment:
Also may want these [extras]( https://github.com/QCoDeS/Qcodes/blob/v0.1.3/setup.py#L28-L29 ). Though that is up to you as a user can always install them if they wish.